### PR TITLE
--exclude option added

### PR DESCRIPTION
**Pull request by paddlesteamer**
One may want to ignore some file types (like image, css) during extraction so I added this parameter.

Example usage goes like this:
python svn_extractor.py --url http://example.com --exclude jpg,jpeg,png,gif

### CODE REVIEW

File: svn_extractor.py [hunk 1]
```diff
@@ -8,13 +8,18 @@
 import traceback
 import re
 
+def getext(filename):
+    name, ext = os.path.splitext(filename)
+    return ext[1:]
```
Comment (anantshri) [inline svn_extractor.py]: i suppose this should be [:-1] to capture last element only and not the entire portion after first element. Example if file name is XYZ.exe.jpg currently the getext will get "exe.jpg" we want "jpg"

Comment (paddlesteamer) [inline svn_extractor.py]: Actually in your example os.path.splitext('XYZ.exe.jpg') returns ('XYZ.exe', '.jpg') so ext becomes '.jpg'. I used [1:] to get rid of '.' at the begining.

Comment (anantshri) [inline svn_extractor.py]: my bad in that case. I was just looking at diff without actually running it. Give me a days time i will get this tested locally and will merge. 

Based on looks everything seems to be in place.

**Comment by anantshri** [comment]
Hi @paddlesteamer 
Looks like i dropped the ball on this one.
i have added the patch via https://github.com/anantshri/svn-extractor/commit/39941be5b56c3865ed7cb8771c8b5a1ab6204299 and https://github.com/anantshri/svn-extractor/commit/0a95ecc41cae6806a4bddea5701b80e32371fc0f Sorry i was not able to directly pull it and advance git juggling is beyond me. so it doesn't reflect as your commit but the feature is added and i have added your name as a contributor.

Thanks for the contribution.

**Pull request closed without merging**